test/e2e/lib/pages/signup/signup-processing-page.js

```diff
@@ -17,18 +17,6 @@ export default class SignupProcessingPage extends AsyncBaseContainer {
 		super( driver, By.css( '.signup-processing-screen__content' ) );
 	}
 
-	async waitForPage() {
```

Comment:
Just to confirm: this was deleted intentionally because nobody is using, right?

Reply:
Searching for `signupProcessingPage` and `SignupProcessPage` shows this method is indeed not being used anymore.

```
test/e2e/lib/flows/sign-up-flow.js
42:             const signupProcessingPage = await SignupProcessingPage.Expect( this.driver );
43:             await signupProcessingPage.waitToDisappear( this.accountName, this.password );

test/e2e/lib/flows/sign-up-step.js
21:             const signupProcessingPage = await SignupProcessingPage.Expect( this.driver );
22:             return await signupProcessingPage.waitToDisappear( blogName, passwordForTestAccounts );
```